packs/boundary_worker/README.md

```diff
@@ -0,0 +1,126 @@
+
+## Table of Contents
+
+<!-- TOC -->
+* [Nomad Pack: (HCP) Boundary Worker](#nomad-pack-hcp-boundary-worker)
+  * [Table of Contents](#table-of-contents)
+  * [Requirements](#requirements)
+  * [Usage](#usage)
+    * [Application](#application)
+    * [Nomad](#nomad)
+    * [Outputs](#outputs)
+  * [Notes](#notes)
+    * [Resource Sizing](#resource-sizing)
+  * [Author Information](#author-information)
+  * [License](#license)
+<!-- TOC -->
+
+## Requirements
+
+- HashiCorp Nomad `1.7.x` or [newer](https://developer.hashicorp.com/nomad/install)
+- HashiCorp Nomad Pack `0.1.0` or [newer](https://releases.hashicorp.com/nomad-pack/)
+- Nomad Task Driver(s) for [`raw_exec`](https://developer.hashicorp.com/nomad/docs/drivers/raw_exec)
+- HCP Boundary Cluster ID, see [Notes](#notes)
+
+## Usage
+
+The `boundary_worker` Pack can be run with a local fileset (commonly used when developing the Pack), or via the [@workloads Nomad Pack Registry](https://github.com/workloads/nomad-pack-registry).
+
+A Pack available _locally_ may be run like so:
+
+```shell
+nomad-pack run ./packs/boundary_worker
+```
+
+A Pack available via the [@workloads Nomad Pack Registry](https://github.com/workloads/nomad-pack-registry) may be run like so:
+
+```shell
+# add the Pack Registry
+nomad-pack registry add workloads github.com/workloads/nomad-pack-registry
+
+# run the Pack
+nomad-pack run boundary_worker --registry=workloads
+```
+
+<!-- BEGIN_PACK_DOCS -->
+
+### Application
+
+This section describes Application-specific configuration.
+
+| Name                                     | Description                                                                                                        | Default |
+| ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------- |
+| app_boundary_helper_output_file_mode     | File Mode of the Output File created by the Boundary Helper binary.                                                | `"0644"` |
+| app_boundary_helper_path                 | Path to the Boundary Helper binary.                                                                                | `"boundary-helper"` |
+| app_cors_allowed_origins                 | Allowed CORS Origins for the Boundary Worker.                                                                      | `["*"]` |
+| app_cors_enabled                         | Toggle to enable CORS support for the Boundary Worker.                                                             | `true` |
+| app_disable_mlock                        | Toggle to disable MLock for the Boundary Worker.                                                                   | `true` |
+| app_enable_hcp_boundary_support          | Toggle to enable HCP Boundary Support (and forego self-hosted Boundary Enterprise Cluster registration workflows.  | `true` |
```

Comment:
Above in the README it says an HCP cluster id is needed, but this seems to imply it would work for self-hosted Boundary too?

Reply:
We technically - from a functional perspective - support both HCP and self-hosted, but right now focus on HCP-first.

I've updated the README to make it more clear